backend/consensus/base.py

```
@@ -558,6 +653,69 @@ async def exec_transaction_with_session_handling(
+                                                        {
+                                                            "hash": transaction.hash,
+                                                            "address": queue_address,
+                                                            "previous_status": previous_transaction[
+                                                                "status"
+                                                            ],
+                                                        },
+                                                        transaction_hash=transaction.hash,
+                                                    )
+                                                )
+                                            except Exception:
+                                                pass
```

Ruff (0.13.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)